appium/tests/specs/live/inbox/CheckReplyAndForwardForEncryptedEmail.spec.ts

```diff
@@ -39,7 +40,7 @@ describe('INBOX: ', () => {
     // check reply all message
     await EmailScreen.clickMenuButton();
     await EmailScreen.clickReplyAllButton();
-    await NewMessageScreen.checkFilledComposeEmailInfo([recipientEmail, senderEmail], replySubject, quoteText);
+    await NewMessageScreen.checkFilledComposeEmailInfo([recipientEmail, senderEmail], replySubject, quoteText, undefined, [ccEmail]);
```

Comment:
One day we'll probably rewrite the method to be clearer like accepting `{to: [...], cc: [...]}` and so on. But for this issue, this looks good.

Reply:
I agree